addons/mail/static/src/models/partner.js

```diff
@@ -19,8 +19,8 @@ registerModel({
          */
         async fetchSuggestions(searchTerm, { thread } = {}) {
             const kwargs = { search: searchTerm };
-            const isNonPublicChannel = thread && thread.model === 'mail.channel' && thread.public !== 'public';
-            if (isNonPublicChannel) {
+            const isGroupRestrictedChannel = thread && thread.model === 'mail.channel' && thread.authorizedGroupFullName;
```

Comment:
This change is not correct, group and chat should also have the restriction.

Reply:
okay, then i will just call it `isNonPublicChannel` since all channel_type is included except for the public channel